src/components/communitySettings/HivemindSettings/TcHivemindSettings.tsx

```
@@ -172,6 +174,23 @@ function HivemindSettings() {
         setIsActivePlatformLoading(false);
         break;
 
+      case 4:
+        setIsActivePlatformLoading(true);
+        const { results: websiteResults } = await retrievePlatforms({
+          name: 'website',
+          community: communityId,
+        });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)